test/VoteManager.js

```diff
@@ -116,7 +115,7 @@ describe('VoteManager', function () {
 
         const stakeBefore = (await stakeManager.stakers(stakerIdAcc3)).stake;
 
-        const votes = [100, 200, 300, 400, 500, 600, 700, 800, 900];
+        const votes = [0];
```

Comment:
why was this modified?

Reply:
No assets to report in the first epoch and votes array cant be empty so went with this.